conan/api/model/list.py

```diff
@@ -48,6 +49,13 @@ def load(file):
             raise ConanException(f"Package list file invalid JSON: {file}\n{e}")
         except Exception as e:
             raise ConanException(f"Package list file missing or broken: {file}\n{e}")
+        if "graph" in content and "command":
+            relative_path = os.path.relpath(file, os.getcwd())
```

Comment:
``relpath`` is problematic and break in windows if the absolute path is a different drive, which happens often in CI environments.
Either use just the filename or a generic one